npm/design-system/src/styles/variables.scss

```diff
@@ -0,0 +1,234 @@
+  chill-20: rgba(164, 197, 244, 1),
+  chill-10: rgba(222, 235, 252, 1),
+  chill-05: rgba(237, 243, 253, 1),
+  olive-60: rgba(93, 100, 12, 1),
+  olive-50: rgba(114, 123, 15, 1),
+  olive-40: rgba(143, 154, 25, 1),
+  olive-30: rgba(211, 228, 27, 1),
+  olive-20: rgba(225, 237, 100, 1),
+  olive-10: rgba(238, 245, 168, 1),
+  olive-05: rgba(246, 250, 209, 1),
+  papaya-60: rgba(143, 68, 10, 1),
+  papaya-50: rgba(190, 90, 14, 1),
+  papaya-40: rgba(235, 107, 10, 1),
+  papaya-30: rgba(242, 141, 64, 1),
+  papaya-20: rgba(246, 175, 121, 1),
+  papaya-10: rgba(252, 229, 212, 1),
+  papaya-05: rgba(253, 241, 231, 1),
+  green-60: rgba(41, 102, 10, 1),
+  green-50: rgba(54, 133, 15, 1),
+  green-40: rgba(68, 164, 20, 1),
+  green-30: rgba(79, 191, 23, 1),
+  green-20: rgba(107, 219, 51, 1),
+  green-10: rgba(204, 244, 185, 1),
+  green-05: rgba(236, 251, 228, 1),
+  cran-50: rgba(228, 28, 95, 1),
+  brand-00: rgba(28, 228, 150, 1),
+  brand-01: rgba(8, 41, 63, 1),
+  accent-00: rgba(177, 99, 255, 1),
+  accent-01: rgba(54, 197, 255, 1),
+  accent-02: rgba(230, 255, 30, 1),
+);
+
+// --- Generate color classes
+// .text- color utilities
+// .bg- color utilities
+
+@each $name, $color in $colors {
+  .text-#{"" + $name} {
+    color: $color;
+  }
+  .bg-#{"" + $name} {
+    background-color: $color;
+  }
+}
+
+// ---
+
+// --- Color Variables
+
+$black: #000000;
+$metal-90: #212426;
+$metal-80: #2d3134;
+$metal-70: #3e4347;
+$metal-60: #535a5f;
+$metal-50: #6b747b;
+$metal-40: #879097;
+$metal-30: #a0a7ac;
+$metal-20: #bec2c6;
+$metal-10: #e6e8ea;
+$metal-05: #f4f5f6;
+$white: #ffffff;
+$red-70: #990b12;
+$red-60: #bf0d16;
+$red-50: #e0101a;
+$red-40: #f13740;
+$chill-90: #082144;
+$chill-80: #0c3164;
+$chill-70: #104289;
+$chill-60: #1556b2;
+$chill-50: #1b6fe4;
+$chill-40: #498eee;
+$chill-30: #76a8ef;
+$chill-20: #a4c5f4;
+$chill-10: #deebfc;
+$chill-05: #edf3fd;
+$olve-60: #5d640c;
+$olive-50: #727b0f;
+$olive-40: #8f9a19;
+$olive-30: #d3e41b;
+$olive-20: #e1ed64;
+$olive-10: #eef5a8;
+$olive-05: #f6fad1;
+$papaya-60: #8f440a;
+$papaya-50: #be5a0e;
+$papaya-40: #eb6b0a;
+$papaya-30: #f28d40;
+$papaya-20: #f6af79;
+$papaya-10: #fce5d4;
+$papaya-05: #fdf1e7;
+$green-60: #29660a;
+$green-50: #36850f;
+$green-40: #44a414;
+$green-30: #4fbf17;
+$green-20: #6bdb33;
+$green-10: #ccf4b9;
+$green-05: #ecfbe4;
+$cran-50: #e41c5f;
+$brand-00: #1ce496;
+$brand-01: #08293f;
+$accent-00: #b163ff;
+$accent-01: #36c5ff;
+$accent-02: #e6ff1e;
+
+// ---
+
+// Typography
+
+// --- Font Families
+
+$font-stack-sans: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif,
+  "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+$font-stack-mono: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+
+// ---
+
+// --- Text Sizes/Scale
+
+$text-xs: 12px;
+$text-s: 14px;
+$text-base: 16px;
+$text-m: 24px;
+$text-l: 32px;
+$text-xl: 40px;
+$text-xxl: 48px;
+$text-xxxl: 64px;
+$text-xxxxl: 76px;
```

Comment:
This is a conversation we need to have early on about sizing. We could use `rem` based sizing everywhere, which means the entire design system is responsive to the browser's text size. This is likely to be harder, and I have no experience in actually implementing it, but it is something to consider.

Reply:
You want a mix of em and rem. Generally for spacing you want rem, but yes, I don't want pixels. This is a good video on it: https://www.youtube.com/watch?v=_-aDOAMmDHI&ab_channel=KevinPowell

Reply:
Agreed. Part of this work was feeling the limitations of exporting direct from figma. This is one of them. I'm for setting the root to `16px` and letting the rems have their fun.